src/index.ts

```diff
@@ -1,21 +1,24 @@
 import 'isomorphic-fetch'
 import { Researcher, Participant } from "./model/index"
 import {
-  APIService,
-  ActivityService,
   ActivityEventService,
+  ActivityService,
   ActivitySpecService,
+  APIService,
   CredentialService,
+  OAuthService,
   ParticipantService,
   ResearcherService,
-  SensorService,
   SensorEventService,
+  SensorService,
   SensorSpecService,
   StudyService,
   TypeService
 } from "./service/index"
-import { Configuration } from "./service/Fetch"
 import { Demo } from "./service/Demo"
+import { Fetch } from "./service/Fetch"
+
+import { decode } from "jsonwebtoken"
```

Comment:
I think we should keep `LAMP-js` clear of the actual OAuth/IdP stuff like JWTs, and instead offer alternate login methods where the API client user is expected to complete the IdP login and already have a long-lived (or short-lived!) token to **provide instead of** a password?

Reply:
That is to say, I think the extra functionality added here might belong in LAMP-dashboard only!

Reply:
For now, we will have API clients log into the dashboard to pull the mindLAMP access token and paste it into their code. We'll think of an OOB flow later.